annotations/tekton-annotations/src/main/java/io/dekorate/tekton/decorator/AddInsecureRegistryParamToTaskDecorator.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2018 The original authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ **/
+
+package io.dekorate.tekton.decorator;
+
+public class AddInsecureRegistryParamToTaskDecorator extends AddParamToTaskDecorator {
+
+  private static final String INSECURE_REGISTRY_PARAM_NAME = "INSECUREREGISTRY";
```

Comment:
```suggestion
  private static final String INSECURE_REGISTRY_PARAM_NAME = "INSECURE_REGISTRY";
```

Reply:
The idea is to use the same params with the equivalent task of the catalog, so I'd rather keep it as is.

Reply:
OK